homeassistant/components/climate/warmup4ie.py

```diff
@@ -0,0 +1,232 @@
+r"""
+platform that offers a connection to a warmup4ie device.
+
+this platform is inspired by the following code:
+https://github.com/alyc100/SmartThingsPublic/tree/master/devicetypes/alyc100/\
+warmup-4ie.src
+
+to setup this component, you need to register to warmup first.
+see
+https://my.warmup.com/login
+
+Then add to your
+configuration.yaml
+
+climate:
+  - platform: warmup4ie
+    name: YOUR_DESCRIPTION
+    username: YOUR_E_MAIL_ADDRESS
+    password: YOUR_PASSWORD
+    location: YOUR_LOCATION_NAME
+    room: YOUR_ROOM_NAME
+
+# the following issues are not yet implemented, since i have currently no need
+# for them
+# OPEN  - holiday mode still missing
+#       - commands for setting/retrieving programmed times missing
+"""
+
+import logging
+import voluptuous as vol
+from homeassistant.components.climate import ClimateDevice, PLATFORM_SCHEMA
+try:
```

Comment:
Remove this `try...except` block.

Reply:
Okay, can do. I only added this block to make the code work with the current stable release. The const.py has been created only a few days ago and a few constants i needed have moved there from the climate __init__.py. Will this stay that way?

Reply:
No matter, changed as suggested. Done.